src/auth/input/auth.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsNotEmpty, IsString } from 'class-validator';
+
+export class CreateUserDto {
+  @IsNotEmpty()
+  @IsString()
+  email: string;
+
+  @IsNotEmpty()
+  @IsString()
+  password: string;
```

Comment:
You might be interested in setting up some constraints, e.g. min and max length of the password. I believe we don't want users to have a 2 character passwords.

Note on max length: while I advocate that we should be able to set a password of length reaching even 10000 characters if we want, we still need some limits (password of **size** 1MB in text is too much). In case of `bcrypt`, the default implementation supports up to 72 characters. You can easily check this by setting e.g. 128-character password, and then try to log in with the first 72 characters - should work. So for starters you could set max password length to 72 characters.